datahub-kubernetes/datahub/charts/datahub-gms/values.yaml

```diff
@@ -151,6 +151,7 @@ readinessProbe:
 # helm install datahub-gms datahub-gms/
 global:
   datahub_analytics_enabled: true
+  graphService: neo4j
```

Comment:
in other configs we seem to use the "_" convention for naming. 

should this be something like graph_service?

Reply:
better yet, graph_service_impl